extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -32,6 +33,20 @@ public final class TransactionManagerConfiguration {
     @ConfigItem(defaultValue = "ObjectStore")
     public String objectStoreDirectory;
```

Comment:
Not related to your PR, but I think you should use `object-store.directory` here, to be consistent with `object-store.datasource`?

```suggestion
    @ConfigItem(name = "object-store.directory", defaultValue = "ObjectStore")
    public String objectStoreDirectory;
```

Reply:
@mmusgrov Do you agree this change ?

Reply:
+1

Reply:
Done.